spec/examples/testscript/testscript_fhirpath_test.json

```diff
@@ -0,0 +1,152 @@
+  },
+  "url": "http://hl7.org/fhir/TestScript/testscript_fhirpath_test",
+  "identifier": {
+    "system": "urn:ietf:rfc:3986",
+    "value": "urn:oid:1.3.6.1.4.1.21367.2005.3.7.9881"
+  },
+  "version": "1.0",
+  "name": "testscript_fhirpath_test",
+  "title": "TestScript FHIR Path test",
+  "status": "draft",
+  "experimental": true,
+  "date": "2017-01-18",
+  "publisher": "HL7",
+  "contact": [
+    {
+      "name": "Support",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "support@HL7.org",
+          "use": "work"
+        }
+      ]
+    }
+  ],
+  "description": "TestScript example resource with simple Patient search test. The read tests will utilize user defined dynamic variables that will hold the Patient search parameter values.",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US",
+          "display": "United States of America (the)"
+        }
+      ]
+    }
+  ],
+  "purpose": "Patient Search Operation",
+  "copyright": "© HL7.org 2011+",
+  "metadata": {
+    "link": [
+      {
+        "url": "http://hl7.org/fhir/patient.html",
+        "description": "Demographics and other administrative information about an individual or animal receiving care or other health-related services."
+      }
+    ],
+    "capability": [
+      {
+        "required": true,
+        "validated": false,
+        "description": "Patient Search Operation",
+        "link": [
+          "http://hl7.org/fhir/http.html#search"
+        ],
+        "capabilities": "http://hl7.org/fhir/CapabilityStatement/example"
+      }
+    ]
+  },
+  "fixture": [
+    {
+      "id": "fixture-patient-bundle",
+      "autocreate": false,
+      "autodelete": false,
+      "resource": {
+        "reference": "fixtures/Patient/example_bundle.json"
+      }
+    },
+    {
+      "id": "fixture-patient-bundle-entry",
+      "autocreate": false,
+      "autodelete": false,
+      "resource": {
+        "reference": "fixtures/Patient/example_bundle_entry.json"
+      }
+    }
+  ],
+  "variable": [
+    {
+      "name": "PatientSearchBundleTotal",
+      "description": "Evaluate the returned Patient searchset Bundle.total value",
+      "expression": "Bundle.total.toInteger()"
+    }
+  ],
+  "test": [
+    {
+      "id": "02-PatientSearchDynamic",
+      "name": "Patient Search Dynamic",
+      "description": "Search for Patient resources using the user defined dynamic variables ${PatientSearchFamilyName} and ${PatientSearchGivenName} and validate response.",
+      "action": [
+        {
+          "assert": {
+            "description": "Confirm Bundle.total.toInteger() >= entry.count()",
+            "expression": "Bundle.total.toInteger() >= entry.count()",
+            "value": "true",
+            "operator": "greaterThan",
```

Comment:
The operator here isn't needed, or should be `equals`.  You have a greater than operator in the expression, but you're asking the TestScript engine to compare the boolean result of the expression to the specified value `true` using equals.  Fine to not support operators other than `equals`, but in this case the operator should be `equals`.

Reply:
That's right. I put "greaterThan" intentionally to test it fails. (as in expected result) Sorry about lack of explanation.